src/lavinmq/config.cr

```diff
@@ -170,15 +181,11 @@ module LavinMQ
         p.on("--clustering-etcd-endpoints=URIs", "Comma separeted host/port pairs (default: 127.0.0.1:2379)") do |v|
           @clustering_etcd_endpoints = v
         end
-        p.on("--default-consumer-prefetch=NUMBER", "Default consumer prefetch (default 65535)") do |v|
-          @default_consumer_prefetch = v.to_u16
-        end
-        p.on("--default-user=USER", "Default user (default: guest)") do |v|
-          @default_user = v
-        end
-        p.on("--default-password=PASSWORD", "Hashed password for default user (default: '+pHuxkR9fCyrrwXjOD4BP4XbzO3l8LJr8YkThMgJ0yVHFRE+' (guest))") do |v|
-          @default_password = v
-        end
+
+        p.separator("\nCommands:")
```

Comment:
```suggestion
        p.separator("\nFlags:")
```
Or something like that, `-h` doesn't feel like a command.

Reply:
Miscellaneous sounds good to me

Reply:
```suggestion
        p.separator("\nMiscellaneous:")
```